storage/range.go

```diff
@@ -433,7 +433,7 @@ func (r *Range) Desc() *proto.RangeDescriptor {
 // SetDesc atomically sets the range's descriptor. This method calls
 // processRangeDescriptorUpdate() to make the range manager handle the
 // descriptor update. Note that processRangeDescriptorUpdate()
-// acquires the metaLock.
+// acquires a lock on the range manager.
```

Comment:
it's not a lock on the range manager (last time I checked). I agree `metaLock` doesn't really tell you anything though, not even which mutex it is which is being held. Or is it not talking about the MetaLock? I'm confused.
Also, isn't the next line in the comment contradicting the previous?